File: Python3/프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    size = list(map(lambda x : sorted(x), sizes))
    w, h = size[0][0], size[0][1];
    for i in range(1, len(size)):
        if size[i][0] > w:
            w = size[i][0]
        if size[i][1] > h:
            h = size[i][1]
    answer = w * h
    return answer

File: Java/프로그래머스/1/86491. 최소직사각형/최소직사각형.java
class Solution {
    public int solution(int[][] sizes) {
        int answer = 0;
        int x = 0, y = 0;
        for(int i = 0; i < sizes.length; i++){
            if(Math.max(sizes[i][0],sizes[i][1]) >= x){
                x = Math.max(sizes[i][0],sizes[i][1]);
            }
            if(Math.min(sizes[i][0],sizes[i][1]) >= y){
                y = Math.min(sizes[i][0],sizes[i][1]);
            }
        }
        return x * y;
    }
}